server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DataTransferRuntimeFlowableTask.java

```diff
@@ -45,6 +45,11 @@
 
 @Slf4j
 public class DataTransferRuntimeFlowableTask extends BaseODCFlowTaskDelegate<Void> {
+    /**
+     * ODC allows importing up to 2 GB of files. We estimate a 5-minute timeout based on transmission
+     * rate of 10 MB/s.
+     */
+    private static final int DEFAULT_GET_IMPORT_FILE_TIMEOUT_MILLIS = 300 * 1000;
```

Comment:
shall we make this parameter editable?

Reply:
Under what circumstances do we need to modify this value?